pulsar-functions/runtime/pom.xml

```diff
@@ -63,6 +63,12 @@
       </exclusions>
     </dependency>
 
+      <dependency>
+          <groupId>org.apache.pulsar</groupId>
+          <artifactId>pulsar-broker-common</artifactId>
+          <version>${project.version}</version>
+      </dependency>
+
```

Comment:
Is this necessary? Broker common will also depend on ZK and bunch of other stuff we don't need in functions. 
If there's anything specific, it can be ported back to `pulsar-common`.